root/etc/profile.d/discovery.sh

```diff
@@ -0,0 +1,8 @@
+# /etc/profile.d/discovery.sh - shell extensions for discovery
+
+# Changes to this file should also be reflected in the setup-extension systemd
+# service file, as systemd doesn't source this file
+
+export LD_LIBRARY_PATH=/opt/extension/lib
+export RUBYLIB=/opt/extension/lib/ruby
+export PATH=$PATH:/opt/extension/bin
```

Comment:
They're unset by default at the moment. I can change it to be an append, but that won't work in the systemd unit below ($PATH .. etc. doesn't work in systemd)